### PR TITLE
chore: mark versions older than the latest major - 3 as deprecated on npm

### PR DESCRIPTION
**Pull request by ghusse**
## Pull Request checklist:

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Create automatic tests
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

File: .releaserc.js [hunk 1]
```diff
@@ -51,5 +51,26 @@ module.exports = {
         packageName: 'forest-express-mongoose',
       }
     ],
+    [
+      "semantic-release-npm-deprecate-old-versions", {
+        "rules": [
+          { 
+            "rule": "supportLatest", 
+            "options": {
+              "numberOfMajorReleases": 2,
```
Comment (arnaudbesnier) [inline .releaserc.js:60]: If we agree to support 3 major versions, I find it a bit misleading to read "2", I would expect to read this config:
```suggestion
              "numberOfMajorReleases": 3,
```
to include the current major version.

I guess it is correctly implemented like this with your package, so it is just a "style" comment about the way the configuration is exposed.

File: .releaserc.js [hunk 2]
```diff
@@ -51,5 +51,26 @@ module.exports = {
...
+              "numberOfMinorReleases": "all",
+              "numberOfPatchReleases": "all"
```
Comment (arnaudbesnier) [inline .releaserc.js:62]: 👍

**Review by arnaudbesnier — COMMENTED** [review]
I left a tiny comment about the way we expose supported versions.

File: .releaserc.js (hunk 1 above)
Comment (ghusse) [reply .releaserc.js:60]: The package is behaving as you guessed, I just made a mistake because I thought we wanted to support 2 versions

**Review by arnaudbesnier — APPROVED** [review]

**Pull request merged**